src/gmp/z.d

```diff
@@ -15,6 +15,18 @@ enum isMpZExpr(T) = (is(Unqual!(typeof(T.eval())) == MpZ)); // which returns an
 enum isLazyMpZExpr(T) = (!is(Unqual!T == MpZ) &&            // exclude direct value
                          isMpZExpr!T);
 
+enum Endianess {
+    host = 0,
+    bigEndian = 1,
+    littleEndian = -1,
+};
+
+enum WordOrder {
+    first = 1,
```

Comment:
What about making this more verbose such as `mostSignificantWordFirst = 1` and `leastSignificantWordFirst = -1`?